package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-responsive-image",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
You wanted just to increase the version, or you were planning to change device.js as well? I can do that myself in like two days.